infrastructure/repository/gorm_init.go

```diff
@@ -27,3 +31,8 @@ func GormInit() (*gorm.DB, error) {
 	}
 	return database, nil
 }
+
+func NewGormDatabase() *GORM {
+	db, _ := GormInit()
+	return &GORM{db: db}
+}
```

Comment:
please rename GORM to Postgres, using GORM might cause problems in the future and be mistaken with gorm library